projector-awt/src/main/kotlin/org/jetbrains/projector/awt/data/AwtComponentAccessor.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * GNU General Public License version 2
+ *
+ * Copyright (C) 2019-2021 JetBrains s.r.o.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+package org.jetbrains.projector.awt.data
+
+import java.awt.Component
+
+/**
+ * A small partial reflection-based version of sun.awt.AWTAccessor which is not accessible due to java modules
```

Comment:
1. That does work I suppose, I'll use AWTAccessor directly then.
2. Actual `setLocation/setSize` methods are on `AWTAccessor.ComponentAccessor` interface. 
3. Because `Component.setSize` does a lot more than just setting fields, including invoking virtual methods, whereas in this specific case the *only* thing that needs to be done is exactly setting fields. See `XWindowPeer.handleConfigureNotifyEvent` for a reference implementation of react-to-window-manager-moving-your-windows code. Doing things otherwise is a change in behavior that produces odd results at times.